### ISSUE TITLE
tfa-nightly pip package currently broken

### ISSUE THREAD

**Issue by jkr26** [tensorflow/addons#703]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux, Debian fork
- TensorFlow version and how it was installed (source or binary): via tfa-nightly pip package dependencies
- TensorFlow-Addons version and how it was installed (source or binary): tfa-nightly, via pip
- Python version: 3
- Is GPU used? (yes/no): no

**Describe the bug**

`tfa-nightly` setup.py doesn't use the pinned `tf-nightly` version `tfa-nightly` is tested with (10/29); this causes the line `import tensorflow_addons` to fail when using `tfa-nightly`.

**Code to reproduce the issue**

(bash):
```
virtualenv --python=python3 test_tfa_nightly
source test_tfa_nightly/bin/activate
pip install tfa-nightly
python -c "import tensorflow_addons"
```

Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**

Opened #701 to mitigate, but I think this points to a deeper issue.

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by seanpmorgan** [comment]
Pin has been released #658 so this should be good. Going forward we'll make sure to update setup.py whenever this happens again. Thanks for reporting!

**Comment by jkr26** [comment]
Awesome!
